### PR TITLE
Various minor edits/clarifications to docs/admin/ docs.

### PR DESCRIPTION
**Pull request by davidopp**
Deleted docs/admin/namespaces.md as it was content-free and the topic is
already covered well in docs/user-guide/namespaces.md

@lavalamp

### CODE REVIEW

**Comment by davidopp** [comment]
rebased

File: docs/admin/salt.md [hunk 1]
```diff
@@ -107,11 +107,11 @@ In addition, a cluster may be running a Debian based operating system or Red Hat
...
+1.  When configuring default arguments for processes, it's best to avoid the use of EnvironmentFiles (Systemd in Red Hat environments) or init.d files (Debian distributions) to hold default values that should be common across operating system environments.  This helps keep our Salt template files easy to understand for editors who may not be familiar with the particulars of each distribution.
 
 ## Future enhancements (Networking)
 
-Per pod IP configuration is provider specific, so when making networking changes, its important to sand-box these as all providers may not use the same mechanisms (iptables, openvswitch, etc.)
+Per pod IP configuration is provider-specific, so when making networking changes, its important to sandbox these as all providers may not use the same mechanisms (iptables, openvswitch, etc.)
```
Comment (lavalamp) [inline docs/admin/salt.md]: s/its/it's/

File: docs/admin/node.md [hunk 2]
```diff
@@ -133,10 +141,10 @@ validate the node by health checking based on the `metadata.name` field: we
 assume `metadata.name` can be resolved. If the node is valid, i.e. all necessary
 services are running, it is eligible to run a Pod; otherwise, it will be
 ignored for any cluster activity, until it becomes valid. Note that Kubernetes
-will keep invalid node unless explicitly deleted by client, and it will keep
+will keep the object for the invalid node unless it is explicitly deleted by the client, and it will keep
 checking to see if it becomes valid.
 
-Currently, there are two agents that interacts with Kubernetes node interface:
+Currently, there are two agents that interacts with the Kubernetes node interface:
```
Comment (lavalamp) [inline docs/admin/node.md]: s/interacts/interact/

File: docs/admin/node.md [hunk 3]
```diff
@@ -133,10 +141,10 @@ validate the node by health checking based on the `metadata.name` field: we
...
 Node Controller and Kube Admin.
```
Comment (lavalamp) [inline docs/admin/node.md]: what the heck is a "Kube Admin"? And kubelet should be in this list.

File: docs/admin/node.md [hunk 4]
```diff
@@ -96,23 +101,26 @@ the following conditions mean the node is in sane state:
...
 
 General information about the node, for instance kernel version, kubernetes version
 (kubelet version, kube-proxy version), docker version (if used), OS name.
-The information is gathered by Kubernetes from the node.
+The information is gathered by Kubelet from the node.
 
 ## Node Management
 
 Unlike [Pods](../user-guide/pods.md) and [Services](../user-guide/services.md), a Node is not inherently
 created by Kubernetes: it is either created from cloud providers like Google Compute Engine,
```
Comment (lavalamp) [inline docs/admin/node.md]: s/from/by/

File: docs/admin/node.md [hunk 5]
```diff
@@ -96,23 +101,26 @@ the following conditions mean the node is in sane state:
...
 or from your physical or virtual machines. What this means is that when
```
Comment (lavalamp) [inline docs/admin/node.md]: s/from/sourced from/ (or taken, whatever)

s/your/your pool of/

File: docs/admin/node.md [hunk 6]
```diff
@@ -96,23 +101,26 @@ the following conditions mean the node is in sane state:
...
 Describes the resources available on the node: CPUs, memory and the maximum
-number of pods that can be scheduled on this node.
+number of pods that can be scheduled onto the node.
 
 ### Node Info
 
 General information about the node, for instance kernel version, kubernetes version
 (kubelet version, kube-proxy version), docker version (if used), OS name.
-The information is gathered by Kubernetes from the node.
+The information is gathered by Kubelet from the node.
```
Comment (lavalamp) [inline docs/admin/node.md]: Kubelet or kubelet? (I've been writing it lowercase; we should standardize.)

File: docs/admin/node.md [hunk 7]
```diff
@@ -96,23 +101,26 @@ the following conditions mean the node is in sane state:
 ]
 ```
 
+If the Status of the Ready condition
+is Unknown or False for more than five minutes, then all of the Pods on the node are terminated.
```
Comment (lavalamp) [inline docs/admin/node.md]: s/./ by the node controller./

File: docs/admin/cluster-troubleshooting.md [hunk 8]
```diff
@@ -21,7 +21,7 @@ certainly want the docs that go with that version.</h1>
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Cluster Troubleshooting
-Most of the time, if you encounter problems, it is your application that is having problems.  For application
+Most of the time, if you encounter problems, it is your application that is the root cause.  For application
```
Comment (lavalamp) [inline docs/admin/cluster-troubleshooting.md]: Wow, that's... bold of us. I would remove this or qualify with a "and your cluster has been set up correctly and passed the conformance test" or some such.

File: docs/admin/admission-controllers.md [hunk 9]
```diff
@@ -111,9 +111,11 @@ so that quota is not prematurely incremented only for the request to be rejected
 
 This plug-in will observe the incoming request and ensure that it does not violate any of the constraints
 enumerated in the ```LimitRange``` object in a ```Namespace```.  If you are using ```LimitRange``` objects in
-your Kubernetes deployment, you MUST use this plug-in to enforce those constraints.
+your Kubernetes deployment, you MUST use this plug-in to enforce those constraints. LimitRanger can also
```
Comment (lavalamp) [inline docs/admin/admission-controllers.md]: s/LimitRanger/LimitRange/

File: docs/admin/admission-controllers.md [hunk 10]
```diff
@@ -111,9 +111,11 @@ so that quota is not prematurely incremented only for the request to be rejected
...
+be used to apply default resource requests to Pods that don't specify any; currently, the default LimitRanger
```
Comment (lavalamp) [inline docs/admin/admission-controllers.md]: here too

**Comment by lavalamp** [comment]
Since this was mostly grammar changes, I graded it pretty harshly ;) feel free to ignore some or all comments.

I still think namespaces are primarily an admin concept, but it's a small enough detail that I'm not going to fight about it.

File: docs/admin/node.md (hunk 4 above)
Comment (davidopp) [inline docs/admin/node.md]: I'm not sure I agree with this one -- it is actually created by Kubernetes, based on information it gets from the cloud provider. So I think "from" is correct.

Comment (davidopp) [inline docs/admin/node.md]: nm, I see later you suggest "taken from" which I agree with

File: docs/admin/node.md (hunk 6 above)
Comment (davidopp) [inline docs/admin/node.md]: I'm going to leave this for a followup PR.

File: docs/admin/cluster-troubleshooting.md (hunk 8 above)
Comment (davidopp) [inline docs/admin/cluster-troubleshooting.md]: Great point, reworded.

File: docs/admin/admission-controllers.md (hunk 9 above)
Comment (davidopp) [inline docs/admin/admission-controllers.md]: Agh, we're totally inconsistent about calling it LimitRange vs. LimitRanger. But to make it locally consistent I'll apply your suggestion.

Comment (davidopp) [inline docs/admin/admission-controllers.md]: Actually, we have multiple uses of both just within this doc. I'm going to leave this for a followup PR.

**Comment by davidopp** [comment]
Implemented most suggestions, PTAL.

The LimitRange/LimitRanger thing has been bugging me for a long time. We use it inconsistently all over the place. I decided not to try to fix it within this doc, but I agree we need to fix it at some point.

File: docs/admin/admission-controllers.md (hunk 9 above)
Comment (lavalamp) [inline docs/admin/admission-controllers.md]: OK

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**